### DISCUSSION TITLE
[@types/node] `dlopen` is missing from `node:process`

### DISCUSSION THREAD

**Discussion by ceztko** [DefinitelyTyped/DefinitelyTyped#65252] [Issues with a @types package]
In a Visual Studio TypeScript console project, which is using `@types/node` 18.15.13, I'm trying to use [`dlopen`](https://nodejs.org/api/process.html#process_process_dlopen_module_filename_flags) from module [`process`](https://nodejs.org/api/process.html), but TypeScript compilation fails with error `Error TS2339 Property 'dlopen' does not exist on type 'Process'`. The `dlopen` function is indeed present in Node and if the compiler is tricked by casting the module `process` to `any` it will lookup the function at runtime and work correctly. It's my understanding that TypeScript typing is coming from DefinetelyTyped definitions and in fact `dlopen` seems missing from [`process.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/process.d.ts).

**Expected behavior**:

The following code should compile correctly in a TypeScript project.

```
import * as proc from 'node:process';
console.log(proc.dlopen);
```

Attached the a simple project that shows that 'dlopen' is indeed avaiable. Must be bootstrapped with `npm i`.
[Test_process_dlopen.zip](https://github.com/DefinitelyTyped/DefinitelyTyped/files/11301187/Test_process_dlopen.zip)

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Added in #69212
